src/OrchardCore/OrchardCore.ResourceManagement.Abstractions/OrchardCore.ResourceManagement.Abstractions.csproj

```diff
@@ -1,12 +1,14 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <RootNamespace>OrchardCore.ResourceManagement</RootNamespace>
     <!-- NuGet properties-->
     <Title>OrchardCore ResourceManagement Abstractions</Title>
-    <Description>$(OCFrameworkDescription)
+    <Description>
+      $(OCFrameworkDescription)
```

Comment:
new line?

Reply:
Apparently, VS is auto formatting the XML files on save. Sorry I missed it. I just fixed this.